rdbi-recipes/src/test/java/com/lithium/dbi/rdbi/recipes/cache/RedisHashCacheTest.java

```diff
@@ -305,7 +304,7 @@ public TestContainer apply(@Nullable String s) {
                 try {
                     return queue.take();
                 } catch (InterruptedException e) {
-                    throw Throwables.propagate(e);
+                    throw new RuntimeException(e);
```

Comment:
Interrupt

Reply:
this is in a unit test; i'm inclined to leave it as an exception.